tests/unit/s2n_cipher_suites_test.c

```diff
@@ -17,6 +17,21 @@
 
 #include "tls/s2n_cipher_suites.h"
 
+static bool is_placeholder_cipher(struct s2n_cipher_suite* cipher_suite) {
+    /* The null cipher suite is just a placeholder, and is not included */
```

Comment:
I would honestly just do this:
```
return (cipher_suite == &s2n_null_cipher_suite) || 
    (cipher_suite == &s2n_equal_preference_group_start) || 
    (cipher_suite == &s2n_equal_preference_group_end)
```

Reply:
I prefer having it in a function just for better readability. Another benefit with this is that I can write readable comments. Maybe in the future there will be more ignorable cipher suites?

Reply:
I didn't say remove it from the function. I'm saying combine all the if-statements and return the result.

Reply:
Then why move it to a function at all? The point of moving this to a function was to separate the if conditions for readability.

Reply:
I like that you separated it into a new function, that logic should be grouped together. But this way is more concise; separating it into different if-statements is unnecessary. 
It's a common code pattern 🤷‍♀️ 
https://github.com/aws/s2n-tls/blob/main/tls/s2n_early_data_io.c#L60

Reply:
I think I actually prefer the multiple lines TBH. It's easier to comment on each condition and read.

Reply:
I'm not willing to die on this hill. Keep it as is.